### PR TITLE
get the new sd package working on the cloud

### PR DESCRIPTION
**Pull request by brandondutra**
added a test that
* runs analysis via bigquery from csv source files
* runs transformations with image columns in DataFlow
* does distributed training on ml engine
* does batch prediction with the cloud service

### CODE REVIEW

File: solutionbox/code_free_ml/mltoolbox/code_free_ml/data/transform_raw_data.py [hunk 1]
```diff
@@ -130,6 +117,12 @@ def parse_arguments(argv):
       type=int,
       default=100)
 
+  parser.add_argument(
```
Comment (qimingj) [inline solutionbox/code_free_ml/mltoolbox/code_free_ml/data/transform_raw_data.py]: Should we also add worker_type?

File: solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+import unittest
+import uuid
+
+from tensorflow.python.lib.io import file_io
+
+CODE_PATH = os.path.abspath(os.path.join(
+    os.path.dirname(__file__), '..', 'mltoolbox', 'code_free_ml', 'data'))
+
+
+class TestCloudServicesTrainer(unittest.TestCase):
```
Comment (qimingj) [inline solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py]: nit: This is more than trainer. So maybe TestCloudServices?

File: solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py [hunk 3]
```diff
@@ -0,0 +1,308 @@
...
+import tempfile
+import unittest
+import uuid
+
+from tensorflow.python.lib.io import file_io
+
+CODE_PATH = os.path.abspath(os.path.join(
+    os.path.dirname(__file__), '..', 'mltoolbox', 'code_free_ml', 'data'))
+
+
```
Comment (qimingj) [inline solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py]: Also, document the environment set up for running the test? At least you need some versions of gcloud, and gcloud auth and default project are required.

File: solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py (hunk 2 above)
Comment (brandondutra) [inline solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py]: done

File: solutionbox/code_free_ml/mltoolbox/code_free_ml/data/transform_raw_data.py (hunk 1 above)
Comment (brandondutra) [inline solutionbox/code_free_ml/mltoolbox/code_free_ml/data/transform_raw_data.py]: done

File: solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py (hunk 3 above)
Comment (brandondutra) [inline solutionbox/code_free_ml/test_mltoolbox/test_cloud_workflow.py]: done

**Pull request merged**